airbyte-integrations/connectors/source-mssql/metadata.yaml

```diff
@@ -29,6 +29,9 @@ data:
     - language:python
   releases:
     breakingChanges:
+      3.0.0:
+        message: "Remapped columns of types: date, datetime, datetime2, datetimeoffset, smalldatetime, and time from `String` to their appropriate Airbyte types. Customers whose streams have columns with the affected datatype must refresh their schema and reset their data."
+        upgradeDeadline: "2023-10-31"
```

Comment:
the upgrade deadline needs to be at least 7 days out.  This date is in the past

Reply:
This is just a placeholder. I'm going to update it to 1 week from the day we send out comms.